Create CODEOWNERS

jamesm-macrometa opened the pull request:
Created a CODEOWNERS file for the DOCS repository.

jamesm-macrometa committed:
Create CODEOWNERS

Created a CODEOWNERS file for the DOCS repository.

diana-macrometa reviewed (approved):
Looks good. We should test it with some minor PRs, and I might take myself off as owner of everything at some point, but let's see where this gets us.

jamesm-macrometa committed:
Update CODEOWNERS

jamesm-macrometa commented:
> Looks good. We should test it with some minor PRs, and I might take myself off as owner of everything at some point, but let's see where this gets us.

FYI I already removed you as a top-level owner and added @astrotars instead.  You and I are only owners of the /docs/ folder.
